debian/control.d/control.jessie.in

```diff
@@ -3,4 +3,4 @@ Depends: libavahi-client3 (>= 0.6.16), libavahi-common3 (>= 0.6.16), libc6 (>= 2
     libtk8.6 (>= 8.6.0), libudev1 (>= 183), libusb-1.0-0 (>= 2:1.0.9), libuuid1 (>= 2.16), libx11-6, libxext6, libxft2 (>> 2.1.1), libxinerama1, libxss1, libzmq3 (>= 3.2.3+dfsg),
     zlib1g (>= 1:1.1.4), python (<< 2.8), python (>= 2.7~), python:any (>= 2.7.5-5~), libboost-python1.55.0, libboost-thread1.55.0, libboost-system1.55.0,
     bc, cgroup-tools, libtk-img (>=1.13), procps, psmisc, python-tk, python-numpy, python-xlib, python-gtkglext1, python-configobj, python-zmq, python-protobuf (>= 2.4.1),
-    python-avahi, python-simplejson, python-pyftpdlib, tclreadline, tcl8.6, tk8.6, uuid-runtime, yapps2-runtime
+    python-avahi, python-simplejson, python-pyftpdlib, tclreadline, tcl8.6, tk8.6, uuid-runtime, yapps2-runtime, libck0
```

Comment:
`dh_shlibdeps` should be able to pick this up automatically, and this can be dropped.  Same for the hunk in `control.stretch.in`.